src/SFA.DAS.AdminService.Web.Tests/Controllers/Gateway/PeopleInControl/PeopleInControlHighRiskTests.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.AdminService.Web.Controllers.Roatp.Apply;
+using SFA.DAS.AdminService.Web.Infrastructure.RoatpClients;
+using SFA.DAS.AdminService.Web.Services.Gateway;
+using SFA.DAS.AdminService.Web.Validators.Roatp;
+using SFA.DAS.AdminService.Web.ViewModels.Roatp.Gateway;
+using SFA.DAS.AssessorService.Api.Types.Models.Validation;
+using SFA.DAS.AssessorService.ApplyTypes.Roatp;
+
+namespace SFA.DAS.AdminService.Web.Tests.Controllers.Gateway.PeopleInControl
+{
+
+    [TestFixture]
+    public class PeopleInControlHighRiskTests
+    {
+        private RoatpGatewayPeopleInControlController _controller;
+        private Mock<IRoatpApplicationApiClient> _applyApiClient;
+        private Mock<IHttpContextAccessor> _contextAccessor;
+        private Mock<IRoatpGatewayPageViewModelValidator> _gatewayValidator;
+        private Mock<IPeopleInControlOrchestrator> _orchestrator;
+        private Mock<ILogger<RoatpGatewayPeopleInControlController>> _logger;
+
+        private string username => "mark cain";
+        private string givenName => "mark";
+        private string surname => "cain";
+        private Guid applicationId = Guid.NewGuid();
+        [SetUp]
+        public void Setup()
+        {
+            _applyApiClient = new Mock<IRoatpApplicationApiClient>();
+            _contextAccessor = new Mock<IHttpContextAccessor>();
+            _gatewayValidator = new Mock<IRoatpGatewayPageViewModelValidator>();
+            _logger = new Mock<ILogger<RoatpGatewayPeopleInControlController>>();
+            _orchestrator = new Mock<IPeopleInControlOrchestrator>();
+
+            var user = new ClaimsPrincipal(new ClaimsIdentity(new Claim[]
```

Comment:
Just a heads up, I think this setup has been moved into a test base class now - might not be merged in yet

Reply:
Thanks for heads up, am changing test to use this